models.js

```diff
@@ -94,4 +94,15 @@ function updateVote(newVote) {
         return result.rows[0].votes
     })
 }
-module.exports={getTopics, getAllEndPoints, getArtcileById, getArticleAndSort, getCommentsForArticle, addCommentForArticle, updateVote}
\ No newline at end of file
+
+function deleteComment(comment_id){
+    if(isNaN(comment_id)){
```

Comment:
PSQL would throw its own error if you tried to delete a comment with an invalid id, but this way is fine too